prepare-cluster/action.yml

```diff
@@ -0,0 +1,88 @@
+name: 'Prepare Cluster'
+description: 'Prepare Cluster'
+inputs:
+  app-slug:
+    description: 'App Slug.'
+    required: true
+    default: ''
+  api-token:
+    description: 'API Token.'
+    required: true
+    default: ''
+  chart:
+    description: 'Path to the helm chart (One of `chart` or `yaml-dir` is required).'
+    required: false
+  yaml-dir:
+    description: 'The directory containing multiple yamls for a Replicated release.'
+    required: false
+  kubernetes-distribution:
+    description: 'Kubernetes distribution of the cluster to provision.'
+    required: true
+  kubernetes-version:
+    description: 'Kubernetes version to provision (format is distribution dependent).'
+    required: true
+  ttl:
+    description: 'Cluster TTL (duration, max 48h)'
+    required: false
+  helm-values:
+    description: 'A Helm values.yaml file to use'
+    required: false
+  helm-chart-name:
+    description: 'The name of the Helm chart to use'
+    required: false
+  kots-config-values:
+    description: 'The KOTS config values to use'
+    required: false
+runs:
+  using: "composite"
+  steps:
+  - name: 'Create Release'
+    id: 'create-release'
+    uses: replicatedhq/compatibility-actions/create-release@v1
+    with:
+      app-slug: ${{ inputs.app-slug }}
+      api-token: ${{ inputs.api-token }}
+      yaml-dir: ${{ inputs.yaml-dir }}
+      chart: ${{ inputs.chart }}
+  - name: 'Create Customer'
+    id: 'create-customer'
+    uses: replicatedhq/compatibility-actions/create-customer@v1
+    with:
+      app-slug: ${{ inputs.app-slug }}
+      api-token: ${{ inputs.api-token }}
+      customer-name: ${{ github.ref_name }}-${{ inputs.kubernetes-distribution }}-${{ inputs.kubernetes-version }}
+      customer-email: ${{ github.ref_name }}@example.com
+      license-type: "test"
+      channel-slug: ${{ steps.create-release.outputs.channel-slug }}
+  - name: Create Cluster
+    id: create-cluster
+    uses: replicatedhq/compatibility-actions/create-cluster@v1
+    with:
+      api-token: ${{ inputs.api-token }}
+      kubernetes-distribution: ${{ inputs.kubernetes-distribution }}
+      kubernetes-version: ${{ inputs.kubernetes-version }}
+      cluster-name: ${{ github.ref_name }}-${{ inputs.kubernetes-distribution }}-${{ inputs.kubernetes-version }}
+      ttl: ${{ inputs.ttl || '' }}
+      timeout-minutes: ${{ inputs.timeout-minutes || 20 }}
```

Comment:
```
Warning: Unexpected input(s) 'timeout-minutes', valid inputs are ['api-token', 'kubernetes-distribution', 'kubernetes-version', 'cluster-name', 'ttl', 'kubeconfig-path', 'export-kubeconfig']
```

Reply:
Related to #14 
There has also been a discussion before if we should expose `timeout-minutes`.